Doc/library/sqlite3.rst

```diff
@@ -250,11 +258,12 @@ Module functions and constants
    *detect_types* parameter and the using custom **converters** registered with the
    module-level :func:`register_converter` function allow you to easily do that.
 
-   *detect_types* defaults to 0 (i. e. off, no type detection), you can set it to
-   any combination of :const:`PARSE_DECLTYPES` and :const:`PARSE_COLNAMES` to turn
-   type detection on. Due to SQLite behaviour, types can't be detected for generated
-   fields (for example ``max(data)``), even when *detect_types* parameter is set. In
-   such case, the returned type is :class:`str`.
+   *detect_types* defaults to 0 (type detection disabled).
+   Set it to any combination of :const:`PARSE_DECLTYPES` and
+   :const:`PARSE_COLNAMES` to enable type detection.
+   Types cannot be detected for generated fields (for example ``max(data)``),
```

Comment:
Is it only related to PARSE_DECLTYPES or to PARSE_COLNAMES too? Does `SELECT max(data) as "max(data) [integer]"` work?

Reply:
I would expect it to work, but I haven't tried.

Reply:
Anyway, I'm not sure that sentence adds real value for the user. IMO, the documentation of the flags says what's needed to know about this.